### PR TITLE
BO: make sure shop list in header is scrollable with lot of items

### PR DESCRIPTION
**Pull request by soee**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This small PR ads 2 new styles to shop list in the backend (in both default and new theme) `overflow-y: auto;` and `max-height: 500px`. This change won't affect any user that have less than 15 shops (14 or less shops list shouldn't render outside our container with `max-height` set to `500px`) on one instance (multistore enabled). So when the list will be much longer (we have atm. ~ 500 shops) thanks to this change shop list will render scrollbar on the rightside that will allow admins to scroll the list. Before this chnage with long list shop users were unable to reach items rendered outside of visible website area. Please check posted link to ticket on Forge where you can see on 2 screenshots how the list behaves without and with proposed chnages.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-3016
| How to test?  | Create 15 or more shops in multistore mode, apply styles added in the commits to admin themes, toggle shop list in backend header when there is less than 15 shops and when there is 15 shops or more.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8426)

### CODE REVIEW

**Comment by xBorderie** [comment]
Thank you for your PR, Marcin!
The Core team will have a look at it soon.

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
Hello @soee,

for me it's ok, let's see if QA team agree with me or not ^^

Mickaël

**Comment by marionf** [comment]
Hello @soee 
I created 50 shops and I don't see the scroll bar
capture du 2017-10-31 09-46-33

**Comment by soee** [comment]
@marionf the changes i have made in this PR are only in themes SASS files so i assume that before each PS version release they are compiled to CSS. When you were testing this chnages did you compiled SASS files to have updated thems styles (CSS files)? If yes are you sure your styles are not loaded from webbrowser cache?

**Comment by mickaelandrieu** [comment]
@marionf ping me on Monday, I can help you

**Comment by marionf** [comment]
Hello @soee 
Yes, I launched npm install and npm run build in admin-dev/themes/new-theme and admin-dev/themes/default and empty all the caches but I don't see the scroll bar

**Comment by Quetzacoalt91** [comment]
We have an issue with that PR, the diff is empty (https://patch-diff.githubusercontent.com/raw/PrestaShop/PrestaShop/pull/8426.patch) because the source repository has been deleted.

As we cannot validate the changes, we have no other choices to close it and reapply the changes on another branch.
Thanks for your understanding

**Comment by soee** [comment]
@marionf @Quetzacoalt91 Sorry, my fault with the source repository. I will create separate PR after checking @marionf problems.

**Comment by Quetzacoalt91** [comment]
Okay, I let you do it then.
For your information, I manually applied the changes and tested on my shop, it works. Let me know once this is done, I will approve the changes.

**Comment by Quetzacoalt91** [comment]
@soee, I made another PR for your changes in #8500. Unfortunately, I couldn't keep your contribution.

**Pull request closed without merging**
